cancer_ai/validator/competition_handlers/base_handler.py

```diff
@@ -11,7 +11,7 @@ class ModelEvaluationResult:
     fpr: any
     tpr: any
     roc_auc: float
-    run_time: float
+    run_time_s: float
```

Comment:
why the _s?

Reply:
it's seconds - good to include metric in file name